build/squire-raw.js

```diff
@@ -2254,17 +2254,33 @@ var onDrop = function ( event ) {
     while ( l-- ) {
         switch ( types[l] ) {
         case 'text/plain':
+        case 'Text': // IE Specific
             hasPlain = true;
             break;
         case 'text/html':
             hasHTML = true;
             break;
         default:
-            return;
+            break;
         }
```

Comment:
> The way it was written, it would return from the function the first time it found a type that was not text or html.

Yes, that was the intention; if there was anything else I presumed it was not a drag-drop of text. Do you have an example where this is not true?

> Really it needs to check the entire list of types to see if any are text or html.

But that's not what this is doing; it's breaking as soon as it finds anything that's not Text/HTML, but there's no guarantee of ordering.